packages/crawler/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "crawler",
+  "version": "0.0.0",
+  "private": true,
+  "license": "UNLICENSED",
+  "scope": "@porsche-design-system",
+  "author": "Dr. Ing. h.c. F. Porsche AG",
+  "scripts": {
+    "start": "ts-node src/index.ts",
+    "start-app": "serve-cdn",
+    "test:e2e": "PORT=3001 jest --colors -c tests/e2e/config/jest.config.ts",
+    "clean": "yarn clean:node_modules && yarn clean:reports",
+    "clean:node_modules": "rm -rf ./node_modules && rm -rf ./yarn-error.log",
+    "clean:reports": "rm -rf ./reports"
```

Comment:
Might be a good idea to call this upon `start`?

Reply:
good idea, changed it